fastani/1.34/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:jammy as app
+
+# for easy upgrade later. ARG variables only persist at build time
+ARG FASTANI_VER="v1.34"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="FastANI"
+LABEL software.version=${FASTANI_VER}
+LABEL description="Fast alignment-free computation of whole-genome Average Nucleotide Identity"
+LABEL website="https://github.com/ParBLiSS/FastANI"
+LABEL license="https://github.com/ParBLiSS/FastANI/blob/master/LICENSE"
+LABEL maintainer="Kelsey Florek"
+LABEL maintainer.email="kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Kutluhan Incekara"
+LABEL maintainer3.email="kutluhan.incekara@ct.gov"
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y wget \
+ unzip \
+ libgomp1 && \
+ apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
What happens if you don't install recommended packages? (--no-install-recommends)
```
RUN apt-get update && apt-get install -y --no-install-recommends \
   unzip \
   libgomp1 && \
   apt-get autoclean && rm -rf /var/lib/apt/lists/*
   ```
   
  Same comment as on other dockerfile